src/agent/src/config.rs

```diff
@@ -234,6 +241,34 @@ fn get_bool_value(param: &str) -> Result<bool> {
     })
 }
 
+// Return the value from a "name=value" string.
+//
+// Note:
+//
+// - A name *and* a value is required.
+// - A value can contain any number of equal signs.
+// - We could/should maybe check if the name is pure whitespace
+//   since this is considered to be invalid.
+fn get_string_value(param: &str) -> Result<String> {
+    let fields: Vec<&str> = param.split('=').collect();
+
+    if fields.len() < 2 {
+        return Err(anyhow!("expected name=value"));
```

Comment:
Should use const `ERR_INVALID_GET_VALUE_PARAM` defined below? 
Here and line 261, 266.

Reply:
@liubin - Historically, only the unit tests used those `const` values. However, I agree - the main code should use them too so I've added an extra commit for this change. ptal.